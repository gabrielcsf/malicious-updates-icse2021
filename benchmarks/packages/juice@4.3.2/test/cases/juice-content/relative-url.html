<html>
<head>
    <link href="3.18.1/build/cssreset/cssreset-min.css" rel="stylesheet">
</head>
<body>
</body>
</html>
