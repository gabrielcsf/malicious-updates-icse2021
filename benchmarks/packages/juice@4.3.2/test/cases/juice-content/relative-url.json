{
    "url": "http://yui.yahooapis.com/"
}