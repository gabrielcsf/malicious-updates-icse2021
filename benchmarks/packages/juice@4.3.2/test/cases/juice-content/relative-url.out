<html style="color: #000; background: #FFF;">
<head>
    
</head>
<body style="margin: 0; padding: 0;">
</body>
</html>
